packages/react-next/etc/react-next.api.md

```diff
@@ -1112,11 +1112,11 @@ export const Toggle: React.FunctionComponent<IToggleProps>;
 export const ToggleBase: React.FunctionComponent;
 
 
+export * from "@fluentui/react-button";
 export * from "office-ui-fabric-react/lib/ActivityItem";
 export * from "office-ui-fabric-react/lib/Announced";
 export * from "office-ui-fabric-react/lib/Autofill";
 export * from "office-ui-fabric-react/lib/Breadcrumb";
-export * from "office-ui-fabric-react/lib/Button";

```

Comment:
we should still export these since it contains a lot more:

```
export * from './BaseButton';
export * from './Button.types';
export * from './Button';
export * from './ActionButton/ActionButton';
export * from './CommandBarButton/CommandBarButton';
export * from './CommandButton/CommandButton';
export * from './CompoundButton/CompoundButton';
export * from './DefaultButton/DefaultButton';
export * from './MessageBarButton/MessageBarButton';
export * from './PrimaryButton/PrimaryButton';
export * from './IconButton/IconButton';
```

Reply:
I think we should still export them but not from next, the idea we talked about a legacy package seems like the correct place for these to go.

Reply:
in /compat.ts probably.